de/surveys/data_privacy_day.lang

```diff
@@ -294,13 +294,13 @@ Meine Privatsphäre zu berücksichtigen, wenn ich einkaufe.
 
 
 # q-25
-;Email Address - We’ll use this to contact you if/when Mozilla has privacy or security information we think you’d find interesting.
-E-Mail-Adresse – Darüber kontaktieren wir Sie, wenn Mozilla Informationen zu Datenschutz oder Sicherheit hat, die Sie interessieren könnten.
+;Email Address (optional) - We’ll use this to contact you if/when Mozilla has privacy or security information we think you’d find interesting.
+E-Mail-Adresse (freiwillig) – Darüber kontaktieren wir Sie, wenn Mozilla Informationen zu Datenschutz oder Sicherheit hat, die Sie interessieren könnten.
```

Comment:
I understand, and there should not be additional updates to this file. I’ve done this change manually, because updating en-US only would have removed your translation.
Also, many thanks for the translations!